src/Language/CSharp.ts

```diff
@@ -338,7 +355,13 @@ class CSharpRenderer extends ConvenienceRenderer {
     }
 
     protected emitUsings(): void {
-        for (const ns of ["System", "System.Net", "System.Collections.Generic"]) {
+        for (const ns of [
+            "System",
+            "System.Collections.Generic",
+            "System.Globalization",
+            "System.Net",
+            "Newtonsoft.Json.Converters"
```

Comment:
This goes into the `NewtonsoftCSharpConverter`

Reply:
Sorry, I mean the `NewtonsoftCSharpRenderer`.  The `CSharpRenderer` doesn’t know anything about Newtonsoft anymore.